src/Controller/MainController.java

```diff
@@ -234,6 +234,8 @@ public static boolean isNumeric(String str) {
 			Double.parseDouble(str);
 		} catch (NumberFormatException nfe) {
 			return false;
+		} catch (NullPointerException nfe) {
```

Comment:
This does not seem related related to the import ordering references in the pull request summary.  Additionally, we previously discussed why this change is not appropriate in this method.

Reply:
I meant to only commit the changes concerning the imports for the Study Planner.